samples/cpp/dbt_face_detection.cpp

```diff
@@ -1,4 +1,4 @@
-#if defined(__linux__) || defined(LINUX) || defined(__APPLE__) || defined(ANDROID)
+#if defined(__linux__) || defined(LINUX) || defined(__APPLE__) || defined(ANDROID) || (defined(_MSC_VER) && _MSC_VER>1700)
```

Comment:
BTW, Header file has check for >=1700. What VS2013 feature is required in sample?